src/insert/spacefinder/rules.ts

```diff
@@ -28,19 +30,21 @@ const minDistanceBetweenInlineAds = isInHighValueSection ? 500 : 750;
 
 const candidateSelector = ':scope > p, [data-spacefinder-role="nested"] > p';
 
+const leftColumnOpponentSelector = ['richLink', 'thumbnail']
+	.map((role) => `[data-spacefinder-role="${role}"]`)
```

Comment:
Splitting out richlink and thumbnail allows us to apply more specific rules here, instead of treating them the same as inline elements